client/src/features/common/LuxRoutes.tsx

```diff
@@ -69,6 +70,21 @@ const LuxRoutes: React.FC = () => {
     setPrevUrl(currentUrl)
   }, [data, isNotAnEntityPage, isSuccess, pathname, prevUrl, routes, search])
 
+  useEffect(() => {
+    const listenResizeEvent = (): void => {
+      if (window.innerWidth < theme.breakpoints.md) {
+        setShowMobileAlert(true)
+      }
+      if (window.innerWidth > theme.breakpoints.md) {
+        setShowMobileAlert(false)
+      }
+    }
+
+    window.addEventListener('resize', listenResizeEvent)
```

Comment:
Mobile alert shows only on resize. It doesn't show when the page is first loaded into a sm viewport.

Reply:
@gigamorph I rectified this in PR #468. In that PR, I also moved the logic to it's own file and use it as a hook.